app/controllers/admin/subcategories_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Admin
+  class SubcategoriesController < BaseCategoriesController
+    before_action :resource, :model
```

Comment:
`before_action`s are used to set some instance variables, performs some checks etc., in current case before actin is unnecessary